s3tests_boto3/functional/test_iam.py

```diff
@@ -1795,3 +1796,290 @@ def test_copy_object_allow_get_src_put_dest():
     eq(etag, response['ETag'])
     response = client.delete_user_policy(PolicyName='AllowGetPolicy', UserName=get_main_user_id())
     eq(response['ResponseMetadata']['HTTPStatusCode'], 200)
+
+@attr(resource='user-policy')
+@attr(method='s3 Actions')
+@attr(operation='Verify Allow and Deny Get Bucket Versioning API using IAM policy for self')
+@attr(assertion='succeeds')
+@attr('user-policy')
+@attr('test_of_iam')
+def test_allow_deny_get_bucket_versioning_iam_policy():
+    client = get_iam_client()
+    s3_client_iam = get_iam_s3client()
+    bucket = get_new_bucket(client=s3_client_iam, name="iam1buk1")
+    s3_client_iam.put_object(Bucket=bucket, Key='iam1buk1obj1', Body='bar')
+    # Perform Get Bucket versioning operation on  “iam1buk1”
+    response = s3_client_iam.get_bucket_versioning(Bucket=bucket)
+    eq(response['ResponseMetadata']['HTTPStatusCode'], 200)
+
+    # Deny Get Bucket Versioning API using IAM policy
+    deny_get_versioning_policy = json.dumps(
+        {"Version": "2012-10-17",
+         "Statement": {
+             "Effect": "Deny",
+             "Action": ["s3:GetBucketVersioning"],
+             "Resource": f"arn:aws:s3:::{bucket}"
+             }
+         }
+    )
+    response = client.put_user_policy(PolicyDocument=deny_get_versioning_policy,
+                           PolicyName='deny_policy_versioning', UserName=get_iam_user_id())
+    eq(response['ResponseMetadata']['HTTPStatusCode'], 200)
+
+    # Perform Get Bucket versioning operation on  “iam1buk1”
+    response = s3_client_iam.get_bucket_versioning(Bucket=bucket)
+    e = assert_raises(ClientError, s3_client_iam.get_bucket_versioning, Bucket=bucket)
+    status, error_code = _get_status_and_error_code(e.response)
+    eq(status, 403)
+    eq(error_code, 'AccessDenied')
+    #Perform Put Bucket versioning on “iam1buk1”
```

Comment:
Use neutral quotation marks
```suggestion
    #Perform Put Bucket versioning on "iam1buk1"
```

Reply:
agreed

Reply:
@archanabpatil let's mark this as resolved once the changes suggested by @ketanarlulkar have been incorporated

Reply:
This is there in multiple places